content/en/software/install-and-run/running-on-linux.md

```diff
@@ -119,7 +119,7 @@ Also note that with local installations, `man medley` will not work.  However, a
 	ubuntu@oio:~$ tar -C <interlisp_directory> -xzf <tar_filepath>
 	```
 
-4. Run Medley
+4. [ ] Run Medley
```

Comment:
Are the square brackets here on purpose?  They give you a checkbox.

Reply:
removed